egs/librispeech/ASR/pruned_transducer_stateless4/train.py

```diff
@@ -625,14 +644,16 @@ def compute_loss(
     y = k2.RaggedTensor(y).to(device)
 
     with torch.set_grad_enabled(is_training):
-        simple_loss, pruned_loss = model(
+        simple_loss, pruned_loss, sym_delay = model(
             x=feature,
             x_lens=feature_lens,
             y=y,
             prune_range=params.prune_range,
             am_scale=params.am_scale,
             lm_scale=params.lm_scale,
             warmup=warmup,
+            delay_penalty=params.delay_penalty,
+            return_sym_delay=params.return_sym_delay,
         )
```

Comment:
To stabilize training, we could apply the penalty when `warmup >= 2.`
```suggestion
        delay_penalty = 0.0 if warmup < 2.0 else params.delay_penalty
        simple_loss, pruned_loss, sym_delay = model(
            x=feature,
            x_lens=feature_lens,
            y=y,
            prune_range=params.prune_range,
            am_scale=params.am_scale,
            lm_scale=params.lm_scale,
            warmup=warmup,
            delay_penalty=delay_penalty,
            return_sym_delay=params.return_sym_delay,
        )
```